tests/run_all_ftp_tests.sh

```diff
@@ -9,13 +9,24 @@ set -e
 echo ""
 echo "*** Running all FTP file tests"
 
-ftp_files="blend.t19z.core.f001.co.grib2 cmc_geavg.t12z.pgrb2a.0p50.f000 \
-WW3_Regional_US_West_Coast_20220718_0000.grib2 WW3_Regional_US_East_Coast_20220717_0600.grib2"
+ftp_files="blend.t19z.core.f001.co.grib2 \
+GLOBAL.grib2.2022103000.0000 \
+hiresw.t00z.arw_5km.f00.hi.grib2 \
+naefs_ge10pt.t12z.pgrb2a.0p50_bcf003 \
+rap.t00z.awp130pgrbf00.grib2 \
+seaice.t00z.grb.grib2 \
+sgx_nwps_CG3_20221117_1200.grib2 \
+cmc_geavg.t12z.pgrb2a.0p50.f000 \
+WW3_Regional_US_West_Coast_20220718_0000.grib2 \
+WW3_Regional_US_East_Coast_20220717_0600.grib2 \
+gdas.t12z.pgrb2.1p00.anl.grib2 \
+flxf2022111712.01.2022111712.grb2 \
```

Comment:
Is there some way to use the list from CMake to get ride of the redundancy here? If not no worries

Reply:
Not easily.